frontend/src/pages/Desk/desk_components/UserPackage.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import {Button, Jumbotron, Input } from "reactstrap";
+import InteractiveUserTable from "./InteractiveUserTable";
+
+import axios from '../../../axiosInstance';
+
+
+class UserPackage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {values: [], packages:[], pk: []}
+    }
+
+
+    componentDidMount() {
+        this.getUserPackages();
+    }
+
+    getUserPackages = () =>{
+        axios
+          .post('/api/packages/user_packages/',{recipient:{username: this.props.user.username}})
+          .then(res => {
+              console.log(res)
+            this.setState({
+                pk: res.data.map(item => item.pk),
+                values: res.data.map((item) => item.quantity - item.num_picked_up),
+                packages: res.data.map((item,ind) => 
+                    [
+                        item.log_time,
```

Comment:
Can you make the time more readable (like how it is displayed on "All Waiting Packages")?